de.dlr.sc.virsat.model.extension.budget.cost.ui/META-INF/MANIFEST.MF

```diff
@@ -13,8 +13,13 @@ Require-Bundle: de.dlr.sc.virsat.model.extension.budget.cost,
  de.dlr.sc.virsat.model,
  de.dlr.sc.virsat.project,
  de.dlr.sc.virsat.project.ui,
- de.dlr.sc.virsat.uiengine.ui
+ de.dlr.sc.virsat.uiengine.ui,
+ org.eclipse.core.commands,
+ org.eclipse.emf.common
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Automatic-Module-Name: de.dlr.sc.virsat.model.extension.budget.cost.ui
 Bundle-Vendor: DLR (German Aerospace Center)
+Import-Package: org.eclipse.gef.commands,
```

Comment:
We should avoid usingimport package for consistency reasons.

Reply:
Nice catch 👍 Cleaned up the import statements.